Purchases/Public/Purchases.swift

```diff
@@ -24,18 +24,18 @@ import StoreKit
 public typealias ReceivePurchaserInfoBlock = (PurchaserInfo?, Error?) -> Void
 
 /**
- Completion block for  ``Purchases/checkTrialOrIntroductoryPriceEligibility(_:completionBlock:)``
+ Completion block for  ``Purchases/checkTrialOrIntroductoryPriceEligibility(_:completion:)``
  */
 public typealias ReceiveIntroEligibilityBlock = ([String: IntroEligibility]) -> Void
```

Comment:
since the typealiases won't be available for obj-c, perhaps we can just remove them? are we getting value out of them?

Reply:
That's a great point, internally, I think they are valuable. I can't remember exactly which at the moment, but one of them wraps another and removing them altogether make it look like a confusing completionblock-ception.

Reply:
makes sense. Maybe we can remove the simpler ones, and make the others internal, if we're not sure they'll add value to the end user?

Reply:
Double checked these, removed most, left only two public ones because they are actually used for our API and helpful IMO.

Reply:
✅